sos/policies/__init__.py

```diff
@@ -900,6 +904,18 @@ def init_kernel_modules(self):
         lines = shell_out("lsmod", timeout=0).splitlines()
         self.kernel_mods = [line.split()[0].strip() for line in lines]
 
+    def get_upload_url(self):
+        """Helper function to determine if we should use the policy default
+        upload url or one provided by the user
+        """
+        return self.upload_url or self.commons['cmdlineopts'].upload_url
```

Comment:
If upload URL will be provided both by policy and on cmdline, won't this prefer the policy one / default one? While cmdline should take precedense?

(and what if the URL will be in `/etc/sos.conf`?)

Reply:
Oops, yep - I had the logic backwards there, thanks.

For `/etc/sos.conf` inclusion this would still be overridden with an actual cmdline arg, and if the URL/user/whatever is specified there, it should definitely override the policy defaults imo.